src/components/sbb-expansion-panel-header/sbb-expansion-panel-header.scss

```diff
@@ -0,0 +1,60 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-expansion-panel-header-cursor: pointer;
+  --sbb-expansion-panel-header-text-color: var(--sbb-color-charcoal-default);
+}
+
+:host([disabled]:not([disabled='false'])) {
+  --sbb-expansion-panel-header-cursor: default;
+  --sbb-expansion-panel-header-text-color: var(--sbb-color-granite-default);
+
+  @include sbb.if-forced-colors {
+    --sbb-expansion-panel-header-text-color: GrayText;
+  }
+}
+
+.sbb-expansion-panel-header {
+  @include sbb.button-reset;
+  @include sbb.text-l--regular;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: var(--sbb-spacing-fixed-6x);
+  width: 100%;
+  padding-block: var(--sbb-spacing-responsive-xs);
+  padding-inline: var(--sbb-spacing-fixed-6x);
```

Comment:
I'd suggest to create css vars for gap and padding and padding-block and padding-inline could be shortened to just padding

Reply:
Done in a3e44f05